core/src/main/kotlin/gropius/model/architecture/Interface.kt

```diff
@@ -15,10 +15,13 @@ import org.springframework.data.neo4j.core.schema.CompositeProperty
     """An interface which is part of a specific ComponentVersion.
     Its semantics depend on the InterfaceSpecification it is specified by, e.g. an Interface can represent a REST API.
     Can be used in Relations and affected by Issues.
-    READ is granted if READ is granted on `component`.
+    READ is granted if READ is granted on `interfaceDefinition`.
     """
 )
-@Authorization(NodePermission.READ, allowFromRelated = ["component"])
+@Authorization(NodePermission.READ, allowFromRelated = ["interfaceDefinition"])
+@Authorization(NodePermission.ADMIN, allowFromRelated = ["interfaceDefinition"])
+@Authorization(ComponentPermission.RELATE_TO_COMPONENT, allowFromRelated = ["interfaceDefinition"])
+@Authorization(ComponentPermission.RELATE_FROM_COMPONENT, allowFromRelated = ["interfaceDefinition"])
```

Comment:
Added it as a comment here but applies everywhere:

The question is with what philosophy we want to handle permissions:
1. Propgate them through to all related nodes (like you did by adding all the allowFromRelated)
2. Or Always checking on the nodes that the permissions are granted on in the mutations

Reply:
Ignore this. Realized it was a dumb question